blogposts/2021/how-to-not-break-a-search-engine-or-what-i-learned-about-unglamorous-engineering.md

```diff
@@ -116,13 +121,13 @@ heavily using my local barebones utility, my coworker
 [@joe](https://twitter.com/jc_unknwon) brought this testing to our CI. It was a
 game changer. Now all of our development was subject to this testing, not just
 my local tinkering. I'd venture that this is probably the most valuable part of
-our search testing infrastructure today. The crazy thing is how rock solid it
+our search testing infrastructure today. The wild thing is how rock solid it
```

Comment:
Crazy is considered non-inclusive language so I've swapped this out, feel free to use another word if you prefer